ouroboros-network/src/Ouroboros/Network/Block.hs

```diff
@@ -421,6 +421,12 @@ instance Serialise (HeaderHash block) => Serialise (Point block) where
   encode = encodePoint encode
   decode = decodePoint decode
 
+instance (Serialise (HeaderHash block), Typeable block) => ToCBOR (Point block) where
+  toCBOR = encodePoint encode
+
+instance (Serialise (HeaderHash block), Typeable block) => FromCBOR (Point block) where
+  fromCBOR = decodePoint decode
+
```

Comment:
It's uncelar to me if this is the "Right Thing":tm:

Reply:
What does the "Right Thing" look like?

Reply:
Setting aside whether it's the "Right Thing", I've tested and it works.  😁

Reply:
This instance also makes me feel uneasy. I _think_ I'm concerned about perhaps needing to version the serialization of a `Point`.

@newhoggy can you point out where this instance is being used? Thanks.

Reply:
See https://github.com/input-output-hk/ouroboros-network/pull/3346#discussion_r710947500